llmfoundry/data/packing.py

```diff
@@ -350,6 +354,7 @@ def auto_packing_ratio(dataloader_cfg: DictConfig,
     # Restore rng state.
     reproducibility.load_rng_state(rng_state)
 
+    log.debug(f'Found packing ratio: {packing_ratio}')
```

Comment:
We're already logging this in dataloader.py. I think you can remove this one. Would be nice to make the log in dataloader.py a `log.debug`